AzureCalling/app/src/main/java/com/azure/samples/communication/calling/activities/JoinCallActivity.java

```diff
@@ -25,6 +28,7 @@
 public class JoinCallActivity extends AppCompatActivity {
     private static final String LOG_TAG = JoinCallActivity.class.getSimpleName();
     private EditText editTextTextMeetingName;
+    private ActivityResultLauncher<Intent> mStartForResult;
```

Comment:
If we handle everything in SetupActivity this should not be needed.

Reply:
ok Wayne